packages/mdc-top-app-bar/index.ts

```diff
@@ -21,39 +21,30 @@
  * THE SOFTWARE.
  */
 
-import MDCTopAppBarAdapter from './adapter';
 import {MDCComponent} from '@material/base/component';
 import {MDCRipple} from '@material/ripple/index';
+import {MDCTopAppBarAdapter} from './adapter';
 import {cssClasses, strings} from './constants';
-import MDCTopAppBarBaseFoundation from './foundation';
-import MDCFixedTopAppBarFoundation from './fixed/foundation';
-import MDCShortTopAppBarFoundation from './short/foundation';
-import MDCTopAppBarFoundation from './standard/foundation';
+import {MDCFixedTopAppBarFoundation} from './fixed/foundation';
+import {MDCTopAppBarBaseFoundation} from './foundation';
+import {MDCShortTopAppBarFoundation} from './short/foundation';
+import {MDCTopAppBarFoundation} from './standard/foundation';
+import {RippleFactory} from './types';
```

Comment:
does this need to belong in the `types` file? It seems to only be used in this file.